Microsoft.Toolkit.Uwp/Helpers/SystemInformation.cs

```diff
@@ -121,6 +122,17 @@ public static async Task LaunchStoreForReviewAsync()
         /// </summary>
         public PackageVersion FirstVersionInstalled { get; }
 
+        /// <summary>
+        /// Gets the previous version of the app that was installed.
+        /// This will be the current version if a previous version of the app was installed
+        /// before using <see cref="SystemInformation"/> or if the app is not updated.
+        /// </summary>
+        public PackageVersion PreviousVersionInstalled 
```

Comment:
```suggestion
        public PackageVersion PreviousVersionInstalled
```
 D:\a\1\s\Microsoft.Toolkit.Uwp\Helpers\SystemInformation.cs(130,55): error SA1028: Code should not contain trailing whitespace [D:\a\1\s\Microsoft.Toolkit.Uwp\Microsoft.Toolkit.Uwp.csproj]